supplierProduct/migrations/0001_initial.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 4.1.3 on 2022-12-14 18:47
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('product', '0002_test_data'),
+    ]
```

Comment:
You should have supplierproduct,supplier in here

also you need to have 0002_test_data for the supplierproduct feature and not the product because its already done